examples/with-next/pages/index.tsx

```diff
@@ -0,0 +1,249 @@
+import { ApiResponseError, Offer, Order } from '@duffel/api'
+import { NextPage } from 'next'
+import { FC, Fragment, useState } from 'react'
+import { Header } from '../components/Header'
+import { Hero } from '../components/Hero'
+import * as Card from '../components/Card'
+import { formatCurrency } from '../utils/currency'
+
+const IndexPage: NextPage = () => {
+  const [offer, setOffer] = useState<Offer | null>(null)
+  const [order, setOrder] = useState<Order | null>(null)
+
+  const onReset = () => {
+    setOffer(null)
+    setOrder(null)
+  }
+
+  return (
+    <div className="page">
+      <Header />
+      <Hero />
+
+      <div className="example-flow">
+        <div className="container">
+          {!offer && <SearchCard onSuccess={(offer) => setOffer(offer)} />}
+          {offer && !order && (
+            <BookingCard offer={offer} onSuccess={(order) => setOrder(order)} />
+          )}
+          {offer && order && <ConfirmationCard order={order} />}
+          {offer && (
+            <button className="example-flow__reset" onClick={onReset}>
+              Start new search
+            </button>
+          )}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+interface SearchCardProps {
+  onSuccess(offer: Offer): void
+}
+
+const SearchCard: FC<SearchCardProps> = ({ onSuccess }) => {
+  const [sort, setSort] = useState<'total_amount' | 'total_duration'>(
+    'total_amount'
+  )
+  const [origin, setOrigin] = useState('JFK')
+  const [destination, setDestination] = useState('LHR')
+  const [isFetching, setIsFetching] = useState(false)
+  const [_errors, setErrors] = useState<ApiResponseError[]>([])
+
+  const fetchCheapestOffer = async () => {
+    setIsFetching(true)
+    const { data, errors } = await (
+      await fetch('/api/search', {
+        method: 'post',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          origin,
+          destination,
+          sort,
+        }),
+      })
+    ).json()
+
+    if (data) {
+      onSuccess(data)
+    } else {
+      setErrors(errors)
+      // eslint-disable-next-line
+      console.log(errors)
```

Comment:
You can use `console.info` which you don't need to disable eslint lines.

Reply:
changed~